### PR TITLE
SONAR-8985 part 1 + SONAR-9040

### PR DESCRIPTION
**Pull request by sns-seb**
**SONAR-8985**
* part 1: add column worker_uuid to CE task tables and populate this new columns (currently with hardcoded value "UNKNOWN")
* part 2 :  will be done by @ehartmann 

**SONAR-9040**
* add column execution_count to CE task tables and populate it
* update peeking code to ignore CE tasks already run twice

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public Boolean call() throws Exception {
 
   private Optional<CeTask> tryAndFindTaskToExecute() {
     try {
-      return queue.peek();
+      return queue.peek("UNKNOWN" /*FIXME provide a real worker uuid*/);
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/taskprocessor/CeWorkerCallableImpl.java]: MAJOR Take the required action to fix the issue indicated by this comment. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v64/MakeCeQueueExecutionCountNotNullableTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MakeCeQueueExecutionCountNotNullableTest {
+
+  private static final String TABLE_CE_QUEUE = "ce_queue";
+
+  @Rule
+  public CoreDbTester db = CoreDbTester.createForSchema(MakeCeQueueExecutionCountNotNullableTest.class, "ce_queue.sql");
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
```
Comment (ehartmann) [inline server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v64/MakeCeQueueExecutionCountNotNullableTest.java]: Is there any hidden reason for adding this ExpectedException ?

Comment (sns-seb) [inline server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v64/MakeCeQueueExecutionCountNotNullableTest.java]: nope, bad copy and paste

File: server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl [hunk 3]
```diff
@@ -561,6 +561,8 @@ CREATE TABLE "CE_QUEUE" (
   "COMPONENT_UUID" VARCHAR(40) NULL,
   "STATUS" VARCHAR(15) NOT NULL,
   "SUBMITTER_LOGIN" VARCHAR(255) NULL,
+  "WORKER_UUID" VARCHAR(40) NULL,
```
Comment (simonbrandhof) [inline server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl]: To be verified, but I think that the table `CE_QUEUE` is truncated on version upgrades. In that case column could be NOT NULL.

**Pull request merged**